.scalafmt.conf

```diff
@@ -1,5 +1,16 @@
-version = 2.0.0
-align = none
+version = 2.5.0-RC3
+project.git = true
+project.excludeFilters = [
+  src/test/resources
```

Comment:
This disables the code formatting in the scala files which are used to compare the expected generated output.